changelogs/fragments/docker_compose-fix-error-message-variable.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - docker_compose - correct variable used in warning message about default IP
```

Comment:
This module is called `docker_container`, not `docker_compose`.

Reply:
Oof. Too many docker things breaking lately. 😊